### PR TITLE
Create AMO Legal landing page

### PR DESCRIPTION
**Pull request by aroybalreid**
Landing page to indicate which policies govern AMO content.

### CODE REVIEW

File: en/AMO Legal landing page [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# AMO Policies
+
+Last Updated March 18, 2024
+{: datetime="2024-03-18" }
+
+[addons.mozilla.org (AMO)](https://addons.mozilla.org/) is an online community that hosts content created by Firefox users and developers.  By participating in this community, you agree to comply with Mozilla’s [Terms of Use for Websites and Communications Services](https://www.mozilla.org/en-US/about/legal/terms/mozilla/), and to be subject to Mozilla’s [Privacy Notice for Websites, Communications, and Cookies](https://www.mozilla.org/en-US/privacy/websites/).
```
Comment (peiying2) [inline en/AMO Legal landing page:6]: Remove `en-US` in both links

File: en/AMO Legal landing page [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+You can choose to contribute extensions, ratings, themes, collections, or other content for other members of the community to use.  You can also upload or otherwise add content to create a user profile on AMO.  Mozilla’s [Acceptable Use Policy](https://www.mozilla.org/en-US/about/legal/acceptable-use/) lays out our rules for user behavior on all Mozilla products and websites, including AMO.  In addition, all extensions, themes, and collections shared on AMO must comply with the Mozilla [Add-on Policies](https://extensionworkshop.com/documentation/publish/add-on-policies/), which include technical, privacy, transparency and related standards for add-ons.
```
Comment (peiying2) [inline en/AMO Legal landing page:8]: Remove en-US in the link

File: en/AMO Legal landing page [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+You can choose to contribute extensions, ratings, themes, collections, or other content for other members of the community to use.  You can also upload or otherwise add content to create a user profile on AMO.  Mozilla’s [Acceptable Use Policy](https://www.mozilla.org/en-US/about/legal/acceptable-use/) lays out our rules for user behavior on all Mozilla products and websites, including AMO.  In addition, all extensions, themes, and collections shared on AMO must comply with the Mozilla [Add-on Policies](https://extensionworkshop.com/documentation/publish/add-on-policies/), which include technical, privacy, transparency and related standards for add-ons. 
+
+When a user shares content that violates these policies, we may remove the content from AMO, and in some cases will also limit or suspend the violator’s account.  If you wish to report violations by another user, you can do so in the following ways:
+For extensions, themes, collections, and user profiles: Visit the listing page and click the “Report” button
```
Comment (peiying2) [inline en/AMO Legal landing page:11]: Insert a line break before listed items.

File: en/AMO Legal landing page [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+For ratings: click the “Flag” link, at the bottom of the review
+
+Add-ons that do not comply with these policies are not permitted, and will be removed from AMO and/or disabled in Firefox if discovered.  To learn more about how Mozilla’s content policies are enforced, you can visit the Content Moderation at Mozilla web page.
+
```
Comment (peiying2) [inline en/AMO Legal landing page:15]: Remove extra line break.

**Review by peiying2 — CHANGES_REQUESTED** [review]
A few general formatting issues

**Review by peiying2 — APPROVED** [review]

**Review by flodolo — APPROVED** [review]

**Pull request merged**